### ISSUE TITLE
ICP implementation bug

### ISSUE THREAD

**Issue by institutohidraulica** [dwoiwode/pyCloudCompareCLI#7]
It seems that on the ICP implementation is a bug in the rot param:
rot.value gives error, since **value** is not an attribute. 

Can be soved defining a class ROT, al giving an object with this "value" attribute:

class ROT(object):
  value = ' NONE'

But is not pythonic.
Moreover ROT is declared in capital letters, and should be in lower case.

Can you confirm?
Thanks!!

**Comment by dwoiwode** [comment]
`ROT` is an Enum defined [here](https://github.com/dwoiwode/pyCloudCompareCLI/blob/aeae6472826f9b5ac5a82380240388749e794a68/pyCloudCompare/pyCloudCompare.py#L92). So if you want to use the function you need to use this object, e.g. `ROT.XYZ` if you want to use "XYZ" as value. If you want to use `"NONE"`, just use `None` instead. It might be an idea to allow string arguments instead of only Enums and convert them in the function.

Did I understand your question correctly? Feel free to provide more information if needed!

**Comment by dwoiwode** [comment]
Do you have any others issues with this? Otherwise I would close this issue. Feel free to ask more questions

**Comment by institutohidraulica** [comment]
Sorry for my late reply. I now understand the declaration of class ROT.
I also tried to use it, but I got an error with teh current implementation:
```
class ROT(Enum):
    XYZ = "XYZ"
    X = "X"
    Y = "Y"
    Z = "Z"
    NONE = "NONE"
```
using the ICP function this way:
`cmd.icp(min_error_diff=1e-7, farthest_removal=True, rot=cc.ROT.NONE)`
gives Unknown or misplaced command: '-ROTNONE'
it seems that the chiet space is missing, tehrefore I made it works chaning the enum declaration from 'NONE' to ' NONE'.
Is this correct?
Thanks again!
